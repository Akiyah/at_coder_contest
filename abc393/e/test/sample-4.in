12 3
1 2 4 8 3 6 12 24 9 18 36 72
