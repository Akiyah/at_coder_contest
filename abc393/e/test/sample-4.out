1
2
4
8
3
6
12
12
9
18
18
18
